tools/sdk-versioner/src/main.rs

```diff
@@ -18,56 +19,102 @@ use toml::value::{Table, Value};
     about = "CLI tool to recursively update SDK/Smithy crate references in Cargo.toml files",
     version
 )]
-struct Args {
-    /// Path(s) to recursively update Cargo.toml files in
-    #[clap()]
-    crate_paths: Vec<PathBuf>,
-
-    /// SDK version to point to
-    #[clap(long)]
-    sdk_version: Option<String>,
-    /// Smithy version to point to
-    #[clap(long)]
-    smithy_version: Option<String>,
-
-    /// Path to generated SDK to point to
-    #[clap(long)]
-    sdk_path: Option<PathBuf>,
+#[allow(clippy::enum_variant_names)] // Want the "use" prefix in the CLI subcommand names for clarity
+enum Args {
+    /// Revise crates to use paths in dependencies
+    UsePathDependencies {
+        /// Root SDK path the path dependencies will be based off of
+        #[clap(long)]
+        sdk_path: PathBuf,
+        /// Path(s) to recursively update Cargo.toml files in
+        #[clap()]
+        crate_paths: Vec<PathBuf>,
+    },
```

Comment:
is this mode actually still needed?

Reply:
It's useful for local development when working on examples.